pkgs/by-name/el/element-call/name.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/package.json b/package.json
+index e25221f2..5268d11a 100644
+--- a/package.json
++++ b/package.json
+@@ -1,4 +1,5 @@
+ {
++  "name": "element-call",
```

Comment:
Please try to upstream this diff

Reply:
I opened a PR [here](https://github.com/element-hq/element-call/pull/2386)